app/src/androidTest/java/de/test/antennapod/service/playback/PlaybackServiceTaskManagerTest.java

```diff
@@ -123,6 +129,42 @@ public void update(EventDistributor eventDistributor, Integer arg) {
         pstm.shutdown();
     }
 
+    @Test
+    public void testQueueUpdatedUponDownloadComplete() throws Exception {
+        final Context c = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        { // Setup test data
+            List<FeedItem> queue = writeTestQueue("a");
+            FeedItem item = DBReader.getFeedItem(queue.get(0).getId());
+            FeedMedia media = new FeedMedia(item, "http://abc.test/acme.mp3", 12345, "audio/mp3");
+            item.setMedia(media);
+            DBWriter.setFeedMedia(media).get();
+            DBWriter.setFeedItem(item).get();
+        }
+
+        PlaybackServiceTaskManager pstm = new PlaybackServiceTaskManager(c, defaultPSTM);
+        final FeedItem testItem = pstm.getQueue().get(0);
+        assertThat("The item is not yet downloaded",
+                testItem.getMedia().isDownloaded(), is(false));
```

Comment:
I think `assertFalse` is more readable than `assertThat(looong line , is(false));`

Reply:
`assertFalse` is more concise. But I find the test output of `assertThat` (or `assertEquals`) are more readable. 

With `assertFalse`, the junit test output doesn't tell you the expected output is false. It just says `AssertionError`. Developers have to go back to the codes to know the expected / actual.  In contrast, `assertThat` / `assertEquals` print out the expected / actual in test output.

Reply:
You can add a message to `assertFalse`, too: `assertFalse("xy should be false", xy)`